src/views/tutorial-view/components/image-config/docs.mdx

```diff
@@ -0,0 +1,39 @@
+---
+componentName: ImageConfig
+---
+
+# ImageConfig
```

Comment:
I'd love to get this into Swingset, but couldn't get the `glob` matching for Swingset's `componentsRoot` option (in `next.config.js`) to work. Probably due to a lack of experience with `glob`!

I would love any input here, for now I just toggled between `componentsRoot` options (loading from `src/views/tutorials-view/components` in order to see this Swingset page).